### ISSUE TITLE
[Feature] Pass extra VMAF args (like n_subsample, pool)

### ISSUE THREAD

**Issue by t-nil** [master-of-zen/Av1an#806]
[ab-av1](https://github.com/alexheretic/ab-av1) has a switch to pass extra ffmpeg libvmaf parameters. I would really love to try out `n_subsample`, as `--probing-rate` gives near unusable results, but the VMAF switch could atleast speed up VMAF calculation at faster presets, where it matters.

**Comment by damian101** [comment]
Right, `--probing-rate` drops frames completely, which messes up the motion component, so it's only a good option at high frame rates if you care about accuracy. I assume the `n_subsample` thing only skips frames for everything except the motion component, which is of course a much better approach.

**Comment by veikk0** [comment]
See also #724.

Users of `n_subsample` should also know about [a common quirk with values that are divisible by 2](https://github.com/Netflix/vmaf/issues/1214).

**Comment by damian101** [comment]
Yes, rav1e has very high frame quality inconsistency, especially every fourth frame is way higher in quality...
